qa/util/src/main/java/io/camunda/qa/util/cluster/TestStandaloneCamunda.java

```diff
@@ -255,30 +261,78 @@ public TestStandaloneCamunda withBrokerConfig(final Consumer<BrokerBasedProperti
    */
   public TestStandaloneCamunda withRecordingExporter(final boolean useRecordingExporter) {
     if (!useRecordingExporter) {
-      brokerProperties.getExporters().remove(RECORDING_EXPORTER_ID);
+      registeredExporters.remove(RECORDING_EXPORTER_ID);
       return this;
     }
-
     return withExporter(
         RECORDING_EXPORTER_ID, cfg -> cfg.setClassName(RecordingExporter.class.getName()));
   }
 
   /**
-   * Adds or replaces a new exporter with the given ID. If it was already existing, the existing
-   * configuration is passed to the modifier. If it's new, a blank configuration is passed.
+   * Enables/disables usage of the elasticsearch exporter using {@link #ELASTICSEARCH_EXPORTER_ID}
+   * as its unique ID.
+   *
+   * @param useElasticsearchExporter if true, will enable the exporter; if false, will remove it
+   *     from the config
+   * @return itself for chaining
+   */
+  public TestStandaloneCamunda withElasticsearchExporter(final boolean useElasticsearchExporter) {
+    if (!useElasticsearchExporter) {
+      registeredExporters.remove(ELASTICSEARCH_EXPORTER_ID);
+      return this;
+    }
+    return withExporter(
+        ELASTICSEARCH_EXPORTER_ID,
+        cfg -> {
+          cfg.setClassName(ElasticsearchExporter.class.getName());
+          cfg.setArgs(Map.of("url", "http://" + esContainer.getHttpHostAddress()));
+        });
+  }
+
+  /**
+   * Enables usage of the camunda exporter using {@link #CAMUNDA_EXPORTER_ID} as its unique ID.
+   *
+   * @return itself for chaining
+   */
+  public TestStandaloneCamunda withCamundaExporter() {
+    withExporter(
+        CAMUNDA_EXPORTER_ID,
+        cfg -> {
+          cfg.setClassName(CamundaExporter.class.getName());
+          cfg.setArgs(
+              Map.of(
+                  "connect",
+                  Map.of("url", "http://" + esContainer.getHttpHostAddress()),
+                  "index",
+                  Map.of("prefix", "global"),
```

Comment:
❓ Was it on purpose that you set the global prefix?

Reply:
I need it because the exporter was trying to update the schema of exiting tasklist/operate indices with the famous
```
Index name: process. Not supported index changes are introduced. Data migration is required. Changes found:
```
but maybe it is been fixed since then?

Reply:
I removed it for now

Reply:
The error above is a known issue with the new Exporter when rerunning the application with existing indices. Should be fixed anytime soon if not already.

Reply:
@panagiotisgts I still have the issue in `main` branch